data/transfers.ts

```diff
@@ -1,52 +1,81 @@
-let TRANSFERRED = [
-    {"name": "Transferred",
-    "value": 7},
-    {"name": "Not Transferred",
-    "value": 66}
-]
+// transfers.ts is updated for 2022
 
-let TERM_TRANSFERRED = [
-    {"name": "First Year",
-    "value": 8},
-    {"name": "Second Year",
-    "value": 13},
-    {"name": "Third Year",
-    "value": 17},
-    {"name": "Fourth Year",
-    "value": 3}
+// What was the last academic term that you completed in SE?
+const LAST_SE_TERM = [{
+    "name": "3A",
+    "value": 1
+}, {
+    "name": "3B",
+    "value": 1
+}, {
+    "name": "4A",
+    "value": 1
+},
 ]
 
-let REASONS_TRANSFERRED = [
-    {"name": "Exchange",
-    "value": 1},
-    {"name": "Flexible Course Requirements",
-    "value": 6},
-    {"name": "Graduate Early",
-    "value": 4},
-    {"name": "Avoid a Course",
-    "value": 6}
+// Why did you switch out?
+const REASONS_TRANSFERRED = [{
+    "name": "Exchange",
+    "value": 0
+}, {
+    "name": "Flexible Course Requirements",
+    "value": 2
+}, {
+    "name": "Graduate Early",
+    "value": 0
+}, {
+    "name": "Avoid a Course",
+    "value": 2
+}, {
+    "name": "Other",
+    "value": 1
+},
 ]
 
-let DISLIKED_COURSES_TRANSFERRED = [
-    {"name": "CS 348",
-    "value": 1},
-    {"name": "ECE 105",
-    "value": 2},
-    {"name": "ECE 106",
-    "value": 2},
-    {"name": "MATH 213",
-    "value": 1},
-    {"name": "SE 380",
-    "value": 1}
+// Was switching out the best choice for you?
+const HAPPY_SWITCHING = [{
+    "name": "Yes",
+    "value": 3
+}, {
+    "name": "No",
+    "value": 0
+},
 ]
-let REGRET_TRANSFFERED = [
-    {"name": "No",
-    "value": 7}
+
+// If you had to start again, what school/program would you enrol in?
+const START_AGAIN_IN_SE = [{
+    "name": "SE",
+    "value": 3
+}, {
+    "name": "Other",
+    "value": 0
+},
 ]
+
+// Are you happy with your choice to come to SE / Waterloo?
+const SWITCHER_SATISFACTION = [{
```

Comment:
not sure what the best graph is for this question

Reply:
Do you think a standard bar graph would work? (Ratings as x, count as y)